### ISSUE TITLE
import/order alphabetize breaks newlines 

### ISSUE THREAD

**Issue by syabro** [benmosher/eslint-plugin-import#1566]
```typescript
import * as React from 'react'
import { partialRight } from 'lodash'

import { BaseSettingsFormFields, WidgetSettingsForm } from 'widgets/common/BaseSettingsForm'
import { WysiwygField } from 'ui/fields/WysiwygField/WysiwygField'

import { WRichText } from './WRichText'
import { WRichTextType } from './constants'
```

if I turn on `alphabetize: {order: 'asc'}` it starts to show errors

```
  1:1   error  There should be at least one empty line between import groups  import/order
  4:1   error  There should be at least one empty line between import groups  import/order
  7:1   error  There should be at least one empty line between import groups  import/order
```

asking to add more new lines

So "correct" code should look like 

```typescript
import { partialRight } from 'lodash'

import * as React from 'react'

import { WysiwygField } from 'ui/fields/WysiwygField/WysiwygField'

import { BaseSettingsFormFields, WidgetSettingsForm } from 'widgets/common/BaseSettingsForm'

import { WRichText } from './WRichText'

import { WRichTextType } from './constants'
```

**Comment by lipe-dev** [comment]
This is happening here as well. It threats every line as a new group, or at least that is what seems to be happening.

**Comment by ljharb** [comment]
This is fixed by #1562, and is a duplicate of #1561. It will be in the next release.

**Comment by syabro** [comment]
@ljharb thanks

**Comment by aequasi** [comment]
Why is that what the correct code should look like...? 


I would expect `'import/order': [2, {'newlines-between': 'always', 'alphabetize': {order: 'asc'}}],` to turn this:

```
import * as React from 'react'
import { partialRight } from 'lodash'

import { BaseSettingsFormFields, WidgetSettingsForm } from 'widgets/common/BaseSettingsForm'
import { WysiwygField } from 'ui/fields/WysiwygField/WysiwygField'

import { WRichText } from './WRichText'
import { WRichTextType } from './constants'
```

Into this:
```
import { partialRight } from 'lodash'
import * as React from 'react'

import { WysiwygField } from 'ui/fields/WysiwygField/WysiwygField'
import { BaseSettingsFormFields, WidgetSettingsForm } from 'widgets/common/BaseSettingsForm'

import { WRichTextType } from './constants'
import { WRichText } from './WRichText'
```
